util/fipstools/delocate/delocate.go

```diff
@@ -520,6 +520,12 @@ func (d *delocation) processAarch64Instruction(statement, instruction *node32) (
 					d.redirectors[symbol] = redirector
 					symbol = redirector
 					changed = true
+				} else if didChange && symbolIsLocal && len(offset) > 0 {
```

Comment:
Is the first inputFile the compiled bcm.c? Is that why it won't have symbols with offsets that were renamed?
Are we sure that the first if will not be hit by those kinds of symbols?

Reply:
> Is the first inputFile the compiled bcm.c?

No, it's an archived file -- `-a $<TARGET_FILE:bcm_c_generated_asm>`

```
./delocate -a $<TARGET_FILE:bcm_c_generated_asm> -o bcm-delocated.S ${BCM_ASM_PROCESSED_SOURCES}
```

> Is that why it won't have symbols with offsets that were renamed?

Case by case, need to check delocate.go. But for local symbol and the statement matching memRef rule, yes. 

> Are we sure that the first if will not be hit by those kinds of symbols?

For the first file, `didChange` is false and the symbol is not change.

Reply:
Thanks for the explanation, Bryce.